### ISSUE TITLE
USER ISSUE: "When attemting to emit Air pollution" in law only affects Powered Cart and Excavator

### ISSUE THREAD

**Issue by neyna** [StrangeLoopGames/EcoIssues#6235]
If you create a low using "When attemting to emit Air pollution", it only works for  Powered Cart and Excavator when you are expecting it to work with blast furnace for example. Because blast furnace emits air pollution.

This should be fixed or you should add something else.

**Comment by LiaungYip** [comment]
This can already be done by making a law like:

WHEN ATTEMPTING TO Craft Item WHERE OBJECT IS (Blast Furnace or Stove or Cement Kiln) THEN TAX $0.10.

We implemented this on a previous server. We called the law "Carbon Tax". :)

**Comment by neyna** [comment]
Yes we did it that way too but it's still not logical and looks like a bug.

**Comment by gabeux** [comment]
+1. Since it takes 24h to make a law, and then once players notice it doesn't work, it takes another 24h to revoke & work around, it can be pretty bad on certain servers.
Thanks for the workaround guys!
